File: beecrowd/1/1010/1010.cs
using System;

class URI {

    static void Main(string[] args) {
        int[] c = new int[2];
        int[] n = new int[2];
        double[] v = new double[2];
        for (int i = 0; i < 2; i++) {
            string[] input = Console.ReadLine().Split(' ');
            c[i] = int.Parse(input[0]);
            n[i] = int.Parse(input[1]);
            v[i] = double.Parse(input[2]);
        }
        Console.WriteLine("VALOR A PAGAR: R$ {0}", (n[0] * v[0] + n[1] * v[1]).ToString("0.00"));
    }
}

File: beecrowd/1/1010/1010.py
c = []
n = []
v = []
for x in range(2):
    line = input().split()
    c.append(int(line[0]))
    n.append(int(line[1]))
    v.append(float(line[2]))

print("VALOR A PAGAR: R$ %.2f" % (n[0] * v[0] + n[1] * v[1]) )
